addons/sale/models/sale_order_line.py

```diff
@@ -543,6 +550,15 @@ def _get_display_price(self):
         """
         self.ensure_one()
 
+        if self.product_type == 'combo':
+            return 0
+        if self.combo_item_id:
+            return self._get_combo_item_display_price()
+        return self._get_display_price_ignore_combo()
+
+    def _get_display_price_ignore_combo(self):
```

Comment:
The display price of a combo line should be 0 on the SO (so that we don't add it to the total price of the SO, which already includes the combo item item prices).

However, in order to compute the display price of a combo item line, we still need the "real" price of the combo product, which is done via `_get_display_price_ignore_combo` (which considers the combo line as a "normal" line).

So `_get_display_price_ignore_combo` is simply a helper method that shouldn't be overridden. It's called in `_get_display_price` to avoid code duplication, but I could've created 2 entirely separate methods. Any overrides of `_get_display_price` should still work fine (and combo lines should still have a 0 display price in those overrides).

I can add a comment for clarification.